dev-qt/prison/prison-5.38.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A barcode api"
+DESCRIPTION="prison has a prison::AbstractBarcode, which is the base class  \
```

Comment:
You missed the important first line of the description...  "A Qt-based barcode abstraction layer/library that provides uniform access to generation of barcodes with data."